Source/Urho3D/Core/CommandLine.h

```diff
@@ -55,7 +55,7 @@ namespace CLI
 namespace detail
 {
 
-template <> constexpr const char *type_name<ea::string>() { return "TEXT"; }
+template <> constexpr std::string type_name<ea::string>() { return "TEXT"; }
```

Comment:
This change breaks the build. Why did you change it?

Reply:
Apperently, MSVC uses the std::string overload and not the "const char*" one,(which is related to the update of CLI11) causing build issues with VS2022.
I haven't tested compilation with other C++ version (that's why I missed the compilation issue), I will check a propriate solution that also works for C++ < 20.